code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -33,18 +34,23 @@
 	var/magic_fluff_string = " You draw the Coder, symbolizing bugs and errors. This shouldn't happen! Submit a bug report!"
 	var/tech_fluff_string = "BOOT SEQUENCE COMPLETE. ERROR MODULE LOADED. THIS SHOULDN'T HAPPEN. Submit a bug report!"
 	var/bio_fluff_string = "Your scarabs fail to mutate. This shouldn't happen! Submit a bug report!"
-
+/*
```

Comment:
Easier to test they dont die immediately on spawning. Knew I'd forget to uncomment eventually